### ISSUE TITLE
How to include images?

### ISSUE THREAD

**Issue by l0b0** [bmcmurray/hekyll#20]
I tried to include an image in a presentation, but it seems difficult. This doesn't work (the alt text shows up instead):

```
![Git Data Transport Commands by Oliver Steele](file://../../_posts/git-transport.png "Git data transport commands")
```

I also tried with `file://_posts/git-transport.png` and `file://git-transport.png`. I can't put the file in `_site` since that is cleaned out whenever files change (with `--auto`) or the server is restarted.
